config/routes.rb

```diff
@@ -189,4 +189,8 @@
     get 'ping' => 'ping#index'
     get 'revision' => 'ping#revision'
   end
+
+  ################################################################################
+  # Incoming Webhook Endpoint
+  resources :sendgrid_events, only: :create, format: false, defaults: { format: :json }
```

Comment:
Any reason not to put this in `api:v1` namespace?

Reply:
It is not public API. I think it is better to not mix those.

Reply:
I was hoping to keep `format: :json` out of UI resources. Anyways, it is fine here too.